src/web/modules/gateway_accounts/createSuccess.njk

```diff
@@ -18,6 +19,25 @@
 A {{ account.type | upper }} Gateway account has been created through Admin Users.
 </p>
 
+{% if isStripe %}
+	<h2 class="govuk-heading-m">What to do next</h2>
+	<p class="govuk-body">In Zendesk, setup the Stripe go live email using the Zendesk macro.  
+	<p class="govuk-body">Replace the placeholders in the macro with the following:</p>
+
+	{{ govukTable({
+			head: [ 
+				{ text: "Zendesk Macro Placeholder"}, 
```

Comment:
The most minor in all the world, this should use sentence case as per https://design-system.service.gov.uk/styles/typography/
```suggestion
				{ text: "Zendesk macro placeholder key"}, 
```

Reply:
Done - I like the level of detail!